### PR TITLE
disable bazel per PR

### PR DESCRIPTION
**Pull request by mikedanese**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1172)

### CODE REVIEW

File: prow/jobs.yaml [hunk 1]
```diff
@@ -38,10 +38,9 @@ kubernetes/heapster:
 
 kubernetes/kubernetes:
 - name: pull-kubernetes-bazel
-  always_run: true
-  context: Jenkins Bazel Build
+  #  context: Jenkins Bazel Build
```
Comment (ixdy) [inline prow/jobs.yaml]: did you mean to comment this out? shouldn't turning off "always_run" and changing the trigger be sufficient?

**Comment by mikedanese** [comment]
@ixdy fixed

**Comment by mikedanese** [comment]
I deployed this change since all builds are going to fail.

**Pull request merged**